nncf/common/graph/graph_matching.py

```diff
@@ -189,7 +225,17 @@ def __and__(self, other):
 
 
 class NodeExpression(Expression):
-    def __init__(self, node_type: str = None, filter_fn=None, node_type_fn: Callable[[dict], str] = None):
+    """A basic Expression that is matched against a single node; the node descriptor is expected to be a dict commonly
```

Comment:
Please, format the docstring according to the Style Guide.

Reply:
Done